ClearKeep/Scenes/ChatGroup/Views/ChatGroupContentView.swift

```diff
@@ -0,0 +1,305 @@
+					self.useFindByEmail = false
+				})
+				.foregroundColor(foregroundCheckmask)
+				.frame(maxWidth: .infinity, alignment: .leading)
+				
+				if useCustomServerChecked {
+					CommonTextField(text: $searchLinkText,
+									inputStyle: $searchLinkStyle,
+									placeHolder: "GroupChat.User.Add.Placeholder".localized,
+									keyboardType: .default,
+									onEditingChanged: { isEditing in
+						searchLinkStyle = isEditing ? .highlighted : .normal
+					})
+				}
+				
+				CheckBoxButtons(text: "GroupChat.AddUserFromEmail.Title".localized, isChecked: $useFindByEmail, action: {
+					self.useCustomServerChecked = false
+				})
+				.foregroundColor(foregroundCheckmask)
+				.frame(maxWidth: .infinity, alignment: .leading)
+				if useFindByEmail {
+					CommonTextField(text: $searchEmailText,
+									inputStyle: $searchEmailStyle,
+									placeHolder: "GroupChat.PasteYourFriendEmail".localized,
+									keyboardType: .emailAddress,
+									onEditingChanged: { isEditing in
+						searchEmailStyle = isEditing ? .highlighted : .normal
+					},
+									submitLabel: .done,
+									onSubmit: searchEmail)
+				}
+				
+				ScrollView(showsIndicators: false) {
+					ForEach($search) { item in
+						UserGroupButton(item.displayName, imageUrl: "", isChecked: self.checkUserIsSelected(item: item.wrappedValue), action: { isSelectedUser in
+							addClient(item.wrappedValue, isSelected: isSelectedUser)
+						})
+					}
+				}
+				RoundedGradientButton(self.useCustomServerChecked ? "GroupChat.Add".localized : "GroupChat.Next".localized, disabled: .constant(self.checkDisableButton()), action: nextToCreateGroup)
+					.frame(maxWidth: .infinity)
+					.frame(height: Constants.heightButton)
+					.font(AppTheme.shared.fontSet.font(style: .body3))
+					.background(backgroundGradientPrimary)
+					.foregroundColor(AppTheme.shared.colorSet.offWhite)
+					.cornerRadius(Constants.cornerRadiusButtonNext)
+					.padding(.horizontal, Constants.spacerTopView)
+					.padding(.bottom, Constants.paddingButtonNext)
+				NavigationLink(
+					destination: CreateGroupView(loadable: $loadable, getProfile: $getProfile, clientInGroup: $addMember),
+					isActive: $isNextCreateGroup,
+					label: {})
+			}
+			.padding(.horizontal, Constants.paddingVertical)
+			.onReceive(inspection.notice) { inspection.visit(self, $0) }
+			.edgesIgnoringSafeArea(.all)
+			.alert(isPresented: self.$isShowAlert) {
+				Alert(title: Text("GroupChat.Warning".localized),
+					  message: Text(self.messageAlert),
+					  dismissButton: .default(Text("GroupChat.Ok".localized)))
+			}.applyNavigationBarPlainStyle(title: "GroupChat.Back.Button".localized,
+										   titleColor: titleColor,
+										   backgroundColors: backgroundButtonBack,
+										   leftBarItems: {
+				BackButtonStandard(customBack)
+			},
+										   rightBarItems: {
+				Spacer()
+			})
+		}
+    }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/ChatGroup/Views/ChatGroupContentView.swift#L137"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>ChatGroupContentView.swift:137</code></td>
    </tr>
  </tbody>
</table>